packages/net/http/_client_connection.pony

```diff
@@ -178,6 +180,7 @@ actor _ClientConnection is HTTPSession
     `_chunk`. This is passed on to the front end.
     """
     _app_handler.finished()
+    _send_pending()
```

Comment:
could you explain why this call is here?
I cannot see how this is related to the _conn state handling.
Isn't this method is called by the HTTPParser when a chunked response body has been fully received?

Reply:
Hi Matthias, 
this allows the state machine to handle the situation that no both pending and unsent requests are empty. If this line is removed then the connection will not close the connection if no work has to be done, will not dispose the connection and waits and waits .... I realize that this is an extra fix but needed it to get the tests working and not timing out.